Sim/SimG4Common/src/EventInformation.cpp

```diff
@@ -47,18 +46,21 @@ void EventInformation::addParticle(const G4Track* aSecondary) {
 
   size_t motherID = aSecondary->GetParentID();
   if (motherID) {  // the particle is not a primary
+    edmParticle.core().status = 201; // FCC convention for status of secondary sim particle
```

Comment:
Hi Valentin, thanks for setting the status. I have a question: why do you set the status exactly to 201? I am not a fan of hard coded numbers somewhere in the source code. Could you please define this convention more visibly, e.g. in the header and document it.